l5-operator/controllers/bestie.go

```diff
@@ -26,11 +26,13 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
-const bestiePort = 8080
+const bestiePort = 8000
 const databaseServiceName = "mysql"
 const databaseName = "bestie"
 const sessionDefaults = "database"
-const bestieImage = "quay.io/rocrisp/cakedemo:v1"
+const bestieImage = "quay.io/skattoju/bestie:v1"
```

Comment:
should be a cr param i think .. can address in a different pr

Reply:
Okay. we can circle back to this.